PyInstaller/depend/utils.py

```diff
@@ -329,6 +329,9 @@ def _restorePaths(old):
         # 'W: library coredll.dll required via ctypes not found'
         if cbin in ['coredll.dll', 'kernel32.dll']:
             continue
+        # On unix ignore the libc
+        if is_unix and (cbin == 'libc.so' or cbin.startswith('libc.so.')):
+            continue
```

Comment:
I was just thinking about this a little bit:
* pyinstaller already excludes all `libc.so*` files and it is defined in the `_unix_excludes` regex patterns in `PyInstaller/depend/dylib.py`.
* Could you try to replace all this code:
```python
if cbin in ['coredll.dll', 'kernel32.dll']:
     continue
# On unix ignore the libc
if is_unix and (cbin == 'libc.so' or cbin.startswith('libc.so.')):
     continue
```
with just this
```python
from .dylib import include_library
if not include_library(cbin):
    continue
```